linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -3201,7 +3201,7 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
       js.Ident(genName(ident.name))(ident.pos)
 
     private def transformMethodIdent(ident: MethodIdent): js.Ident =
```

Comment:
Ah, that was tempting, but `genApply` does not give a more precise `pos` to the method name `Ident`. Therefore, doing that would worsen the source map for code generated by `genNormalApply`.

Reply:
Oh wow. Right. That's quite subtle. Do you think it would make sense to provide an overload of `genApply` that takes a `MethodIdent` to avoid this pitfall?

Reply:
If you want but it will have exactly one call site?

Reply:
Yeah, probably overkill indeed. Let's just leave it like this.